content/SCALE/SCALEUIReference/TopToolbar/Alerts/AlertSettingsScreen.md

```diff
@@ -8,123 +8,148 @@ tags:
 {{< expand "Click here for more information" >}}
 
-![AlertSettingsSharing](/images/SCALE/22.02/AlertSettingsSharing.png "Sharing Alert Settings")
+![AlertSettingsSharing](/images/SCALE/22.12/AlertSettingsSharing.png "Sharing Alert Settings")
 
-Options cover IP addresses bound to an iSCSI ports not found, NFS services not bound to specific IP addresses using 0.0.0.0, NFS share references hosts that cannot b resolved, NTLMv1 authentication attempted in the last 24hours, SMB1 connections to TrueNAS server performed in last 24 hous and share unavailable because it uses a locked dataset.
+You can customize alert settings for when a deprecated service is running, IP addresses bound to an iSCSI portal are not found, NFS services cannot bind to specific IP addresses using 0.0.0.0, and the system cannot resolve NFS share references hosts.
+
+You can also customize alerts for when NTLMv1 authentication has attempted in the last 24 hours, SMB1 connections to TrueNAS server have performed in the last 24 hours, and a share is unavailable because it uses a locked dataset.
 {{< /expand >}}
+
 ### Storage Alert Settings
 **Storage** alert settings apply to quotas, pools, snapshots, and scrub processes on your TrueNAS.
 {{< expand "Click here for more information" >}}
 
-![AlertSettingsStorage](/images/SCALE/22.02/AlertSettingsStorage.png "Storage Alert Settings")
+![AlertSettingsStorage](/images/SCALE/22.12/AlertSettingsStorage.png "Storage Alert Settings")
 
-Options cover critical quota exceeded on dataset, new feature flags available for pools, pool space usage above 80% or 90%, pool status not healthy, quota exceeded on dataset, paused scrub, too many snapshots exist and too many snapshots exist for dataset.
+You can customize alert settings for when a dataset exceeds standard and critical quotas, a pool has new available feature flags, pool space usage exceeds 70, 80, or 90 percent, and pool status is not healthy. 
+
+You can change alert settings for when a pool consumes USB disks, a scrub pauses, and too many snapshots exist.
 {{< /expand >}}
+
 ### System Alert Settings
-**System** alert settings apply to system processes, system dataset, TrueCommand API Key, SSH logins, system reboots, updates and the web interface.
+**System** alert settings apply to system processes, the system dataset, TrueCommand API Key, SSH logins, system reboots, updates, and the web interface.
 {{< expand "Click here for more information" >}}
 
-![AlertSettingsSystem1](/images/SCALE/22.02/AlertSettingsSystem1.png "System Alert Settings")
+![AlertSettingsSystem](/images/SCALE/22.12/AlertSettingsSystem.png "System Alert Settings")
 
-![AlertSettingsSystem2](/images/SCALE/22.02/AlertSettingsSystem2.png "System Alert Settings")
+You can customize alert settings for when Admin is overriden, the boot pool is unhealthy, the system dataset has core files, a device slows down pool I/O, NTP health checks fail, and TrueCommand API keys are disabled or need confirmation.
```

Comment:
I don't think we need to capitalize admin here, and is this the admin user? maybe this if this is what is meant:
```suggestion
You can customize alert settings for when the admin user is overriden, the boot pool is unhealthy, the system dataset has core files, a device slows down pool I/O, NTP health checks fail, and TrueCommand API keys are disabled or need confirmation.
```